docs/modules/ROOT/partials/nav-reference.adoc

```diff
@@ -3,5 +3,6 @@
 * xref:commodore:ROOT:reference/commands.adoc[Commands]
 * xref:commodore:ROOT:reference/cli.adoc[Command Line Options]
 * xref:commodore:ROOT:reference/hierarchy.adoc[Inventory Hierarchy]
+* xref:commodore:ROOT:reference/parameters.adoc[Commodore-managed inventory parameters]
```

Comment:
The referenced page doesn't exist in this PR or in master?

Reply:
fixed